### ISSUE TITLE
python problems

### ISSUE THREAD

**Issue by MyerSmith** [siversalih/pyChrome#1]
Can't get it running under archlinux

```
python2
Python 2.7.13 (default, Dec 21 2016, 07:16:46) 
[GCC 6.2.1 20160830] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyChrome import PyChrome
>>> browser = PyChrome()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pyChrome.py", line 95, in __init__
    self.__start(ghostmode=self.ghost)
  File "pyChrome.py", line 158, in __start
    self.driver = webdriver.Chrome(file_directory,0,chrome_options,None,None,None)
  File "/usr/lib/python2.7/site-packages/selenium/webdriver/chrome/webdriver.py", line 62, in __init__
    self.service.start()
  File "/usr/lib/python2.7/site-packages/selenium/webdriver/common/service.py", line 74, in start
    stdout=self.log_file, stderr=self.log_file)
  File "/usr/lib/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
OSError: [Errno 8] Exec format error
>>> 

```

```
python
Python 3.6.0 (default, Jan 16 2017, 12:12:55) 
[GCC 6.3.1 20170109] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyChrome import PyChrome
CRITICAL:root:urllib2 module is not installed...Exiting program.

```
tried to install the missing package for urllib2 but no success

https://www.archlinux.org/packages/?sort=&q=urllib

**Comment by huyanhh** [comment]
It works when I use Python 2 instead of Python 3

**Comment by siversalih** [comment]
It works with python 2.7 and now it supports macOS and Windows-based platform. For Linux, I have not yet added anything. I'm still working on bringing all the available features from what's currently available on new Selenium 3.4 to pyChrome. So please check back for more updates.

**Comment by siversalih** [comment]
To setup Selenium Python environment, check installation tutorial for [macOS](https://www.youtube.com/watch?v=kizSRWlnPvE&index=1&list=PL8R5Fi8yjw7mNAgRZpQWxJ_ndcL4Tcn0C) or [Windows](https://www.youtube.com/watch?v=3rguo2u3NqM&list=PL8R5Fi8yjw7mNAgRZpQWxJ_ndcL4Tcn0C&index=2).
